content/intermediate/265_spinnaker_eks/testing-helm.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Testing Helm-Based Pipeline"
+weight: 60
+draft: false
+---
+
+We now have Spinnaker Service up and running, we will now loginto Spinnaker UI.
+
+#### Clone Application Git Repo
+```
+cd ~/environment
+git clone https://github.com/aws-containers/eks-microservice-demo.git
+cd eks-microservice-demo
+```
+
+#### Spinnaker UI
+
+##### Create application
+Click on `Create Application` and enter details as `Product-Detail`
+![Spinnaker](/images/spinnnaker/proddetail.png)
+
+##### Create Pipeline
+Click on `Pipelines` under `test-application` and click on `Configure a new pipeline` and add the name as below.
+![Spinnaker](/images/spinnnaker/helmpipeline.png)
+
+##### Setup Trigger
+Click on `Configuration` under `Pipelines` and click on `Add Trigger`. This is the ECR registry we had setup in Spinnaker manifest in [Configure Artifact](/265_spinnaker_eks/configure_artifact/) chapter. Follow the below setup and click on "Save Changes".
+![Spinnaker](/images/spinnnaker/trigger.png)
+
+##### Setup Bake Stage
+- Click on `Add Stage` and select `Bake Manifest` from the dropdown 
+- Select "create new artifact" for **Expected Artifact** and select the Git account that shows in dropdown. This is the Git account we had setup in Spinnaker manifest in [Configure Artifact](/265_spinnaker_eks/configure_artifact/) chapter. And then enter the below git location.
```

Comment:
The dropdown only shows "Define a new manifest..." but I'm guessing that's what you meant by "create new artifact". It would be helpful here to distinguish that one **Expected Artifact** is for Template Artifact and the other **Expected Artifact** is in Overrides. That was in the picture, but wasn't clear from the written description.

Reply:
Added clarification to this section